kit/popups/src/UIForeground/Columns/SecondaryColumn.tsx

```diff
@@ -14,8 +22,22 @@ export function SecondaryColumn({ children, onPress, disabled, negative }: Secon
         };
     }, [disabled, negative, onPress]);
 
+    const isValid = useCheckChildren(
```

Comment:
So it's still will run even in `production`?

Reply:
Sorry, I did it on wrong branch 😄